### DISCUSSION TITLE
set defaultValue for input type='file'

### DISCUSSION THREAD

**Discussion by hiimnhan** [react-hook-form/react-hook-form#4432] [Q&A]
Hi, I don't know how to set defaultValue for input type=file. The situation is I have a form and I fill some values to this for Edit function. But I don't know how the input type=file can show the exist file (which is an URL from AWS). Please show me a way! Thank you.

**Comment by bluebill1049** [comment]
pretty sure you can only set the reference (name of the file)

**Reply by hiimnhan** [reply]
@bluebill1049 So there is no way to get the old file of the input to filled in it?
